<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>1066</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_title">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="5,2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_nowsee"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_show"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3,2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_devicecontroller"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_systemlog"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
